user-manual/getting-started/quick-start.md

```diff
@@ -179,7 +179,7 @@ vanus-display-**************     1/1     Running   0          12s
 ```shell
 vsctl subscription create \
   --eventbus quick-start \
-  --sink 'http://quick-display:80' \
+  --sink 'http://quick-display.vanus:80' \
```

Comment:
remove `.vanus` or modify other `--sink`  in order to have the same style

Reply:
remove `.vanus`